### ISSUE TITLE
u03-tf-ethics Revision

### ISSUE THREAD

**Issue by lhumphrey3511** [kcarnold/cs344-exam-23sp#36]
@kangk9908 
In T/F question u03-tf-ethics, 

The original question is as follows: 
"True or false: In the context of machine learning, fairness is always objective and can be determined by statistical measures alone."

Unfortunately, the question seemed oddly worded and took me ~3 reads to understand it. The word "fairness" is vague. What do you mean by fairness? A lack of harmful bias, perhaps? 

I would rephrase it as follows:

In the context of machine learning, bias can always be measured via statistical measures. 
    a. True 
    b. False

**Comment by kangk9908** [comment]
I see where the confusion comes from and will make the changes!
